docs/pelion_ready/device-management-config.md

```diff
@@ -2,47 +2,47 @@
 - SOTP - the address and size of the SOTP regions in flash used to store the device special key that decrypts the credentials storage.
 - Bootloader - the bootloader image, application and header offset.
 
 Except for connectivity, the majority of the configuration is shared between the application and bootloader, which ensures the bootloader can correctly find, verify, authorize and apply an update to the device application.
 
-For full documentation about bootloaders and firmware update, please read the following documents:
+For more details about bootloaders and firmware update, read the following documents:
 
 - [Introduction to Mbed OS bootloaders](../porting/bootloader.html).
 - [Creating and using an Mbed OS bootloader](../tutorials/bootloader.html).
 - [Bootloader configuration in Mbed OS](../reference/bootloader-configuration.html).
-- [Mbed Bootloader for Device Management](https://github.com/ARMmbed/mbed-bootloader), or the short introduction [in the Device Management documentation](https://cloud.mbed.com/docs/current/updating-firmware/bootloaders.html).
+- [Mbed bootloader for Device Management](https://github.com/ARMmbed/mbed-bootloader), or the short introduction [in the Device Management documentation](https://www.pelion.com/docs/device-management/current/updating-firmware/bootloaders.html).
 - [Updating devices with Mbed CLI](../tools/cli-update.html).
 
-To hasten this process, you can copy the configuration from the [application example](https://github.com/ARMmbed/pelion-ready-example/blob/master/mbed_app.json) as the basis for your application configuration.
+To speed up this process, you can copy the configuration from the [application example](https://github.com/ARMmbed/pelion-ready-example/blob/master/mbed_app.json) as the basis for your application configuration.
 
-## 1. Application configuration
+## Application configuration
 
 Edit the `mbed_app.json` file, and create a new entry under `target_overrides` with the target name for your device:
 
-- **Connectivity** - Specify the default connectivity type for your target. It's essential with targets that lack default connectivity set in `targets.json` or for targets that support multiple connectivity options. For example:
+- **Connectivity** - Specify the default connectivity type for your target. It is essential with targets that lack default connectivity set in `targets.json` or for targets that support multiple connectivity options. For example:
 
    ```
             "target.network-default-interface-type" : "ETHERNET",
    ```
 
    The possible options are `ETHERNET`, `WIFI` and `CELLULAR`.
 
-   Depending on connectivity type, you might have to specify more config options. For an example, please see the defined `CELLULAR` targets in `mbed_app.json`.
+   Depending on connectivity type, you may have to specify more options. For an example, see the defined `CELLULAR` targets in `mbed_app.json`.
```

Comment:
Please see earlier comment about "please".

Reply:
Not willing to be that polite.